app/controllers/email_signups_controller.rb

```diff
@@ -1,32 +1,38 @@
 class EmailSignupsController < PublicFacingController
   def new
-    feed_url = email_signup_params['feed']
-
-    if feed_url.match?(%r{/government/organisations/(.*)\.atom$})
-      redirect_to email_alert_frontend_signup(feed_url)
+    if non_finder_url
+      redirect_to email_alert_frontend_signup
+    elsif feed_url.match?(%r{/government/publications\.atom(.*)$})
+      redirect_to feed_url.sub('.atom', '')
+    elsif feed_url.match?(%r{/government/announcements\.atom(.*)$})
+      redirect_to feed_url.sub('.atom', '')
+    elsif feed_url.match?(%r{/government/statistics\.atom(.*)$})
+      redirect_to feed_url.sub('.atom', '')
```

Comment:
We can probably reduce these 3 conditionals to a single one: `%r{/government/(publications|announcements|statistics)\.atom}` - we shouldn't need to worry about after the atom extension.

Reply:
Yup good point no2